controllers/progressivesync_controller_test.go

```diff
@@ -1127,6 +1254,56 @@ func createApplications(ctx context.Context, targets []Target) ([]argov1alpha1.A
 	return apps, nil
 }
 
+// createApplicationSet creates an ApplicationSet object with labels for the Application template
+func createApplicationSet(ctx context.Context, name string, namespace string, labels map[string]string) (applicationset.ApplicationSet, error) {
```

Comment:
I'm not sure about this one, but I _think_ it makes it a little more clear. Thoughts?
```suggestion
func createApplicationSetWithLabels(ctx context.Context, name string, namespace string, labels map[string]string) (applicationset.ApplicationSet, error) {
```